api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -1257,4 +1257,18 @@ message TrackClusterStats {
   // <config_cluster_manager_cluster_stats_request_response_sizes>`  tracking header and body sizes
   // of requests and responses will be published.
   bool request_response_sizes = 2;
+
+  // If true, some stats will be emitted per-endpoint, similar to the stats in admin ``/clusters``
```

Comment:
this is just for stats sinks, right? the admin `/stats` endpoint will not use this bit, though maybe `/stats/prometheus` will?

Reply:
No, this affects both stat sinks and admin endpoint